node_modules/col-categories/tests/util.js

```diff
@@ -19,12 +19,15 @@ var assert = require('assert');
 /**
  * Assert that a category has all expected properties
  *
- * @param  {Category}           category                The category to assert the properties for
- * @param  {Category}           [expectedCategory]      The category to which the provided category should be compared
- * @param  {Boolean}            [expectAssetCount]      Whether the number of assets associated to the category are expected to be included
- * @throws {AssertionError}                             Error thrown when an assertion failed
+ * @param  {Category}           category                      The category to assert the properties for
+ * @param  {Object}             [opts]                        Optional parameters to verify the category with
+ * @param  {Category}           [opts.expectedCategory]       The category to which the provided category should be compared
+ * @param  {Boolean}            [opts.expectAssetCount]       Whether the number of assets associated to the category are expected to be included
+ * @throws {AssertionError}                                   Error thrown when an assertion failed
  */
```

Comment:
We were starting to have too many of these test utility methods with too many optional arguments, causing the calls to become quite difficult to read and manage. Instead of doing `assertCategory(category, category2, true)`, we can now do `assertCategory(category, {'expectedCategory': category2, 'expectAssetCount': true})`. Even though this is a bit longer, it is a lot more readable and makes adding additional arguments a lot easier.